dev/com.ibm.ws.kernel.boot.core/resources/com/ibm/ws/kernel/boot/resources/LauncherOptions.nlsprops

```diff
@@ -153,21 +191,29 @@ option-desc.archive=\
 #------------------------------\n at 72 chars -- leading tab-----------\n\#
 option-key.include=\
 \ \ \ \ --include=value,value,...
-option-desc.include=\
-\tA comma-delimited list of values. The valid values vary depending on \n\
-\tthe action.
+option-desc.javadump.include=\
+\tA comma-delimited list of values. The valid values are: heap, system \n\
+\tthread.
+option-desc.package.include=\
+\tA comma-delimited list of values. The valid values are: all*, usr,   \n\
+\tminify, runnable. If you specify runnable the resulting package will \n\
+\tbe an executable jar that will run the server. The minify option     \n\
+\twill produce the smallest image possible for the server. The usr     \n\
+\toption creates a package that contains the server, and application   \n\
+\tand not the runtime. The default all option produces a server package\n\
+\tthat contains everything.
+#------------------------------\n at 72 chars -- leading tab-----------\n\#
```

Comment:
This comma placement is confusing me a little. Is this right? 
"The usr option creates a package that contains the server and application, but not the runtime."

I think it'd look like this. I just moved some stuff around so the character counts should be correct in each line.

+\twill produce the smallest image possible for the server. The usr     \n\
+\toption creates a package that contains the server and application,   \n\
+\tbut not the runtime. The default all option produces a server package\n\
+\tthat contains everything.